src/main/java/mate/academy/dao/impl/MovieSessionDaoImpl.java

```diff
@@ -66,7 +66,11 @@ public MovieSession add(MovieSession movieSession) {
     @Override
     public Optional<MovieSession> get(Long id) {
         try (Session session = HibernateUtil.getSessionFactory().openSession()) {
-            return Optional.ofNullable(session.get(MovieSession.class, id));
+            return session.createQuery("FROM MovieSession ms "
+                    + "LEFT JOIN FETCH ms.movie "
+                    + "LEFT JOIN FETCH ms.cinemaHall "
```

Comment:
```suggestion
                    + "JOIN FETCH ms.movie "
                    + "JOIN FETCH ms.cinemaHall "
```
в даному випадку можна використовувати inner join